docker-compose-e2e-statics.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+services:
+  nginx-server:
+    build:
+      context: .
+      dockerfile: ./dockerfiles/nginx-server/Dockerfile
+    ports:
+      - 8081:80
+    networks:
+      - network1
+
+  playwright:
+    build:
+      context: .
+      dockerfile: ./dockerfiles/playwright/Dockerfile
+    command: yarn run test:visual:ci
+    environment:
+      BASE_URL: "http://nginx-server"
+      ipc: host
```

Comment:
What is the issue resolve `ipc=host`?

Reply:
Experiment shows that it's now required :)